src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = {new Bulldozer(),new Truck(),new Excavator()};
+        for (Machine list:machines) {
```

Comment:
Машина - это не лист.
```suggestion
        for (Machine machine : machines) {
```